src/Inciter/AMR/edge.h

```diff
@@ -0,0 +1,80 @@
+#ifndef AMR_edge_t_h
+#define AMR_edge_t_h
+
+#include <iostream>
+#include <ostream>
+
+class edge_t {
+    using edge_ = std::pair<size_t, size_t>;
```

Comment:
Minor thing: Let's use `m_` for member data to make it consistent with the rest of the code. Is that disgusting?